examples/misc/lib/language_tour/classes/employee.dart

```diff
@@ -14,9 +13,8 @@ class Person {
 
 // #docregion method-then-constructor
 class Employee extends Person {
-  // #enddocregion ,
+  // #enddocregion ''
   Employee() : super.fromJson(getDefaultData());
-  // ···
```

Comment:
This is an example of a manually inserted plaster. This sort of thing was necessary because plasters used to be turned on/off for an entire file. This is no longer the case.